src/react/databrowser/objects/ObjectList.jsx

```diff
@@ -77,9 +77,10 @@ export default function ObjectList({ objects, bucketNameParam, prefixParam }: Pr
     }, useFilters, useSortBy);
 
     return <L.ListSection>
-        <T.SearchContainer>
-            <T.ExtraButton icon={<i className="fas fa-plus" />} text="Upload" variant='info' onClick={() => dispatch(push(`/buckets/${bucketNameParam}/upload-object`))} size="default" type="submit" />
-        </T.SearchContainer>
+        <T.ButtonContainer>
+            <T.ExtraButton icon={<i className="fas fa-upload" />} text="Upload" variant='info' onClick={() => dispatch(push(`/buckets/${bucketNameParam}/upload-object`))} size="default" type="submit" />
+            <T.ExtraButton icon={<i className="fas fa-plus" />} text="Create folder" variant='info' onClick={() => dispatch(openFolderCreateModal())} size="default" type="submit" />
```

Comment:
```suggestion
            <T.ExtraButton icon={<i className="fas fa-plus" />} text="Create Folder" variant='info' onClick={() => dispatch(openFolderCreateModal())} size="default" type="submit" />
```

Reply:
AWS uses `Create folder` but for consistency, I agree with you and will change it to `Create Folder`.